src/pages/Landing.tsx

```diff
@@ -69,6 +69,16 @@ const Landing = () => {
               />
             </dd>
           </Dli>
+          <Dli>
+            <dt>Want to know what we’ve been up to?</dt>
```

Comment:
I think the thing we want to focus the reader's attention on is the changes, not the team.  Also the "we" carries a slight danger of exclusion -- "we" are in here doing things, while "you" are out there reading about what we're doing -- whereas we'd rather phrase everything in a way that leaves the door open to participation.

So maybe just "See recent improvements to the PDC data viewer" or something like that?  I can help wordsmith collaboratively if you'd like, but I also trust your judgement now that I've said my piece above!

Reply:
@kfogel How about simply "Recent improvements" as the title?

Reply:
@reefdog Sounds good to me!